### ISSUE TITLE
Game does not work after using dialog blocks

### ISSUE THREAD

**Issue by Deltxa** [polydes/dialog#2]
Here's my error log:
TypeError: Error #1009: Cannot access a property or method of a null object reference.
	at dialog.core::DialogStyle/tieExtensionsToDialogBox()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\engine-extensions\com.polydes.dialog\src\dialog\core\DialogStyle.hx:45]
	at dialog.core::DialogBox()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\engine-extensions\com.polydes.dialog\src\dialog\core\DialogBox.hx:98]
	at dialog.core::Dialog$/globalCall()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\engine-extensions\com.polydes.dialog\src\dialog\core\Dialog.hx:176]
	at scripts::SceneEvents_0/init()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\games-generated\game\Source\scripts\SceneEvents_0.hx:79]
	at com.stencyl.behavior::Behavior/initScript()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\behavior\Behavior.hx:109]
	at com.stencyl.behavior::BehaviorManager/initScripts()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\behavior\BehaviorManager.hx:99]
	at com.stencyl::Engine$/initBehaviors()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\Engine.hx:1271]
	at com.stencyl::Engine/loadScene()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\Engine.hx:1170]
	at com.stencyl::Engine/begin()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\Engine.hx:961]
	at com.stencyl::Engine()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\com\stencyl\Engine.hx:749]
	at Universal/preloaderComplete()[C:\Program Files\Stencyl\plaf\haxe\lib\stencyl\1,00\Universal.hx:408]
	at lime.app::_Event_Void_Void/dispatch()
	at scripts::StencylPreloader/unload()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\games-generated\game\Source\scripts\StencylPreloader.hx:341]
	at scripts::StencylPreloader/onLoaded()[C:\Users\rooar\AppData\Roaming\Stencyl\stencylworks\games-generated\game\Source\scripts\StencylPreloader.hx:310]
	at lime.app::_Event_Void_Void/dispatch()
	at lime.utils::Preloader/start()
	at lime.utils::Preloader/updateProgress()
	at lime.utils::Preloader/current_onEnter()

Basically whenever I test the scene/project right after the first loading section it will always give me this bug. I haven't seen anything that can fix it, and I'm pretty clueless of what the issue is. None of the form posts I've seen have fixed my issue, atleast the ones I knew I could try and did try.

**Comment by justin-espedal** [comment]
What does your dialog style’s extension list look like? Do you have any blank spaces in it like in this screenshot? If you do, delete the blank space and try testing again.

![](https://media.discordapp.net/attachments/227870283749523459/943773167669903370/unknown.png)

**Comment by Deltxa** [comment]
Do you mean the tab you end up with when you click Data Structures (not clicking open structure editor)? If so then in the data panel, there is nothing. Just a blank area.
![image](https://user-images.githubusercontent.com/28176864/158496537-c7b0148b-76b0-4c62-875a-6ee618489112.png)

**Comment by justin-espedal** [comment]
Ah ok, that's the problem then. You need a dialog style to display dialog. If none is specified, it'll use the style named `Default Style`.

It can be kind of a pain to totally set up for the first time, so you might want to copy it from the sample game, as suggested in the [getting started guide](https://dialogextension.com/guides/index.php?f=getting_started#How%20to%20Enable%20the%20Extension%20in%20Your%20Game).

You can grab the sample game from this repository, or you can download it [here](http://polydes.com/3.4.0-extensions/sample-games/dialog-extension-sample.stencyl).

**Comment by Deltxa** [comment]
I tried and it still does not show up.

**Comment by justin-espedal** [comment]
Was the game closed when you copied the `extras` folder?

If you'd like to chat, I'm available on Stencyl's [discord](https://www.stencyl.com/chat/).

**Comment by Deltxa** [comment]
I'm pretty sure it might have been open. Also, it's okay, we can stay here.

**Comment by justin-espedal** [comment]
Close the game and open it again. The new files won't be read while the game is open.

**Comment by Deltxa** [comment]
I've got them to work but the icons are all question marks. Is this normal?
![image](https://user-images.githubusercontent.com/28176864/158504315-1361f34d-a453-420c-8717-166669e538af.png)

**Comment by Deltxa** [comment]
Nevermind, sorted it out! Thank you for helping me!
